tests/approaches/test_oracle_approach.py

```diff
@@ -596,6 +600,35 @@ def test_oracle_approach_painting():
         assert policy_solves_task(policy, test_task, env.simulate)
 
 
+def test_oracle_approach_repeated_nextto_painting():
+    """Tests for OracleApproach class with RepeatedNextToPaintingEnv."""
+    utils.reset_config({
+        "env": "repeated_nextto_painting",
+        "num_train_tasks": 3,
+        "num_test_tasks": 3,
+        "sesame_task_planning_heuristic": "hff"
```

Comment:
interesting, it doesn't work unless you use hff instead of lmcut?

Reply:
It does also work with lmcut. We switched it.